src/main/java/edu/kis/vh/nursery/IntArrayStack.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery;
+
+public class IntArrayStack {
+	private static final int STACK_CAPACITY = 12;
+
+	private static final int EMPTY_STACK_ERROR = -1;
+
+	private int[] numbers = new int[STACK_CAPACITY];
+
+	private int total = EMPTY_STACK_ERROR;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)